webui/react/src/pages/ExperimentDetails/CodeViewer/CodeViewer.tsx

```diff
@@ -120,6 +121,29 @@ const CodeViewer: React.FC<Props> = ({
   runtimeConfig: _runtimeConfig,
 }) => {
   const resize = useResize();
+  const firstConfig = useMemo(() =>
+    _submittedConfig ? Config.submitted : Config.runtime, [ _submittedConfig ]);
+  const configForExperiment = (experimentId: number): SettingsConfig => ({
+    applicableRoutespace: '/experiments',
+    settings: [
+      {
+        defaultValue: firstConfig,
+        key: 'filePath',
+        storageKey: 'filePath',
+        type: { baseType: BaseType.String },
+      },
+      {
+        defaultValue: firstConfig,
+        key: 'fileName',
+        storageKey: 'fileName',
+        type: { baseType: BaseType.String },
+      },
+    ],
+    storagePath: `selected-file-${experimentId}`,
```

Comment:
How come do you store path and name? I think just storing `filePath` is enough. 
Also, we want to have KV data type like key is experiment id, value is path, and stored in selected-code-file something like that.

Reply:
Also we dont store these data in db?